conf/distro/nilrt.inc

```diff
@@ -142,7 +142,7 @@ NILRT_FEEDS_URI_RELEASE            ?= "${NILRT_FEEDS_URI}/${NILRT_FEED_NAME}"
 NILRT_MACHINE_FEED_URI_x64         ?= "${NILRT_FEEDS_URI_RELEASE}/x64"
 NILRT_MACHINE_FEED_URI_xilinx-zynq ?= "${NILRT_FEEDS_URI_RELEASE}/arm"
 
-NILRT_MAIN_FEED_VERSION            ?= "2021.8"
```

Comment:
The purpose of this variable was to let us provide a sane pre-release default version for the `ni-` feeds, that external builders could use before we publish the new feeds. I'm fine with this solution, but we should add a step to the hardknott README telling external builders to go find the latest release on `download.ni.com` and set this variable in their local.conf.

@chaitu236 Could you put up a PR for that.

Reply:
Done https://github.com/ni/nilrt/pull/162